tests/cases/compiler/controlFlowLetVar.ts

```diff
@@ -0,0 +1,119 @@
+// @strictNullChecks: true
+
+declare let cond: boolean;
+
+function f1() {

```

Comment:
It would be nice if these functions had names that showed what they were testing.